lib/main.dart

```diff
@@ -20,7 +23,7 @@ class Concord extends StatelessWidget {
         title: 'Concord',
         // For testing, change the assignment of home to be whatever screen you're working on
         // Example: home: AuthScreen()
-        home: HomeScreen(),
+        home: AuthWidget(),

```

Comment:
While this was good for testing, we're going to use the `HomeScreen` as the default screen;
```suggestion
        home: HomeScreen(),
```